src/Verilator.cpp

```diff
@@ -160,7 +160,8 @@ static void process() {
     //
     V3Assert::assertAll(v3Global.rootp());
 
-    if (!v3Global.opt.xmlOnly()) {
+    if (!(v3Global.opt.xmlOnly() &&
+          !v3Global.opt.flatten())) {
```

Comment:
We put &&'s first on the next line - run "make clang-format" and all the style stuff will clean up for you automatically.

Reply:
Done. I've included the changes to src/Verilator.cpp, but excluded a bunch of other changes in the codebase.